### PR TITLE
Update the front matter for post

### PR DESCRIPTION
**Pull request by abstractj**
I noticed that some documentation like this one http://staging-aerogearsite.rhcloud.com/docs/specs/aerogear-crypto/ was broken. This is the fix.

### CODE REVIEW

**Comment by abstractj** [comment]
@andresagalante @lfryc would you mind to review?

**Comment by andresgalante** [comment]
@abstractj you are missing `section: docs` on your front matter.

**Comment by andresgalante** [comment]
@abstractj also don't add a h1 title. It will be added automatically from the front matter title.
https://gist.github.com/andresgalante/132c9ef91f1a009ffd82

**Comment by abstractj** [comment]
@andresgalante updated. Regarding the HTML tags, I will leave as is to not mess up with what you guys already done

File: docs/specs/aerogear-client-pipe.html [hunk 1]
```diff
@@ -734,7 +734,8 @@
 </div>
 <div id="content">
 <div class="paragraph"><p>---
-layout: basic
```
Comment (andresgalante) [inline docs/specs/aerogear-client-pipe.html]: @abstractj this is strange. Why do we need front matter here?

**Comment by abstractj** [comment]
@andresgalante if we don't, I can just remove it. I just replaced all the front matters like we discussed, nothing specific.

**Comment by lfryc** [comment]
thanks for cleanup :heart_eyes: 

:eyes: reviewing...

**Comment by lfryc** [comment]
I'm preparing a new PR which I will send to yours for review

File: docs/specs/aerogear-client-pipe.html (hunk 1 above)
Comment (lfryc) [inline docs/specs/aerogear-client-pipe.html]: I believe someone just accidentally commited HTML generated from `aerogear-client-pipe.asciidoc`. I'll remove it.

File: docs/specs/aerogear-client-pipe/rest-adapter.asciidoc [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ---
```
Comment (lfryc) [inline docs/specs/aerogear-client-pipe/rest-adapter.asciidoc]: I can't see this document linked from the Site, is it? If it should be removed, let's remove it, if not, let's link it.

File: docs/specs/aerogear-client-pipe/rest-adapter.html [hunk 3]
```diff
@@ -3,131 +3,132 @@
   <head>
```
Comment (lfryc) [inline docs/specs/aerogear-client-pipe/rest-adapter.html]: Again, IMO this is just wrongly commited HTML.

File: docs/specs/aerogear-client-pipe/usage-examples.asciidoc [hunk 4]
```diff
@@ -1,5 +1,6 @@
 ---
```
Comment (lfryc) [inline docs/specs/aerogear-client-pipe/usage-examples.asciidoc]: This is likely replaced by `docs/specs/aerogear-client-paging-usage/index.markdown`

File: docs/specs/aerogear-client-pipe/usage-examples.html [hunk 5]
```diff
@@ -3,75 +3,76 @@
   <head>
```
Comment (lfryc) [inline docs/specs/aerogear-client-pipe/usage-examples.html]: Again HTML generated from asciidoc, remove it?

File: docs/specs/aerogear-security-offline/index.md [hunk 6]
```diff
@@ -1,5 +1,6 @@
 ---
-layout: basic
+layout: post
+section: docs
 title: Offline Storage Specification (OSS)
```
Comment (lfryc) [inline docs/specs/aerogear-security-offline/index.md]: I can't see Offline Storage Specification linked from Site Docs

File: docs/specs/aerogear-security-otp.asciidoc [hunk 7]
```diff
@@ -1,5 +1,6 @@
 ---
-layout: basic
+layout: post
+section: docs
 title: AeroGear Security OTP Specification
```
Comment (lfryc) [inline docs/specs/aerogear-security-otp.asciidoc]: I can't see `AeroGear Security OTP Specification` linked from Site Docs

File: docs/specs/aerogear-sync-use-cases.asciidoc [hunk 8]
```diff
@@ -1,5 +1,6 @@
 ---
-layout: basic
+layout: post
+section: docs
 title: AeroGear Data Sync
```
Comment (lfryc) [inline docs/specs/aerogear-sync-use-cases.asciidoc]: I can't see `AeroGear Data Sync` linked from Site Docs

**Comment by lfryc** [comment]
I found some more deficiencies such as pages that are not accessible from Site indexes (Docs/Guides/..).

**Comment by lfryc** [comment]
Some of my concerns were partially addressed here: https://github.com/abstractj/aerogear.org/pull/4

File: docs/specs/aerogear-security-offline/index.md (hunk 6 above)
Comment (lfryc) [inline docs/specs/aerogear-security-offline/index.md]: My preliminary check `ack aerogear-security-offline` revealed there is really no link here, this spec  should be linked from Docs, right?

File: docs/specs/aerogear-security-otp.asciidoc (hunk 7 above)
Comment (lfryc) [inline docs/specs/aerogear-security-otp.asciidoc]: This spec is linked from `docs/guides/AeroGear-OTP.asciidoc`, I believe we should also make it available from Docs index.

File: docs/specs/aerogear-sync-use-cases.asciidoc (hunk 8 above)
Comment (lfryc) [inline docs/specs/aerogear-sync-use-cases.asciidoc]: Preliminary check `ack aerogear-sync-use-cases` found that this is not really linked.

Remove it OR  link it from Docs index?

**Comment by lfryc** [comment]
Thanks for diving into this @abstractj , I can see there are much more issues now, but at least they got onto surface so we can deal with them :-)

**Comment by lfryc** [comment]
@abstractj btw this needs to be rebased

**Comment by abstractj** [comment]
@lfryc yay, done!

**Comment by lfryc** [comment]
@abstractj I suggest to merge this and address the pages that are not linked properly ( https://github.com/aerogear/aerogear.org/pull/530#discussion_r25937118 ) in another PR/thread/issue, okay?

**Comment by abstractj** [comment]
@lfryc +1 to merge

**Pull request merged**